packages/p2p/src/components/buy-sell/popup.jsx

```diff
@@ -52,6 +59,13 @@ class Popup extends Component {
 
     render() {
         const { ad, handleClose } = this.props;
+        const initial_values = { amount: ad.min_available };
+        const is_buyer = ad.type === 'buy';
+
+        if (!is_buyer) {
+            initial_values.contact_info = ad.contact_info;
+            initial_values.payment_info = ad.payment_info;
+        }
```

Comment:
IMO something like this is more readable:

```suggestion
        const initial_values = {
            amount: ad.min_available,
            ...(ad.type !== 'buy' && {
                contact_info: ad.contact_info,
                payment_info: ad.payment_info,
            }),
        };
```

Reply:
That makes sense, will add these changes. 👍